src/ray/pubsub/test/subscriber_test.cc

```diff
@@ -82,7 +82,8 @@ class MockWorkerClient : public pubsub::SubscriberClientInterface {
       new_pub_message->set_sequence_id(sequence_id);
     }
     reply.set_publisher_id(publisher_id.empty() ? publisher_id_ : publisher_id);
-    callback(status, reply);
+    auto copied = reply;
+    callback(status, std::move(copied));
```

Comment:
Is it possible to avoid the manual copy by supporting both

```
callback(status, reply) // The reply is still needed, don't move, make a copy
```

and 

```
callback(status, std::move(reply)) // The reply is no longer needed, can move
```

Reply:
I think the purpose is to *force* a move. If we support both const& and &&, we can always forget to move and accidentally make copies. So I'd stay with explicit copies.